docker/Dockerfile-ntg

```diff
@@ -0,0 +1,23 @@
+# Use the official Node.js image as the base image
```

Comment:
can we remove all non PR related changes?

Reply:
sure thing , do you want me to revert these changes in another commit on the same branch ? or do you want me to remove them from my branch ?